release-notes/v1.2.0.md

```diff
@@ -0,0 +1,24 @@
+# HPE CSI Driver for Kubernetes 1.2.0 Release Notes
+
+## HPE CSI Driver for Kubernetes 1.2.0
+
+| **Version:** |1.2.0|
+|--------------|-----|
+| **Revision:** | Monday June 15, 2020 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v1.2.0.
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v1.2.0.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-878|csi.k8s|Resource Quotas for RWX PVC's with NFS resources|HPE CSI Driver will create NFS Provisioner deployment for each RWX PVC. These deployments will consume CPU/Memory resources depending on I/O load|We recommend users to set Resource Quota on the namespace where NFS resources are provisioned(default hpe-nfs) to avoid NFS provisioner pods consuming high CPU/Memory. Also, users can set max CPU/Memory limits for each NFS Provisioner pod using StorageClass Parameters nfsResourceLimitsCpuM and nfsResourceLimitsMemoryMi|
+|CON-931|csi.k8s|Re-deployment of pods created on RHCOS fails on RHEL node|Known Issue with a mixture of RHEL and RHCOS worker nodes. https://access.redhat.com/solutions/4582401. XFS filesystem created with RHCOS is not compatible with RHEL 7.|None|
```

Comment:
I think in workaround section, instead of None, we should say our driver doesn't support combination of RHEL7/CoreOS nodes with XFS filesystem due to known RedHat limitation.